src/gui_common/MainMenu.tscn

```diff
@@ -290,7 +290,6 @@ mouse_filter = 1
 size_flags_horizontal = 3
 size_flags_vertical = 6
 theme = ExtResource( 2 )
-text = "x.y.z"
```

Comment:
I kinda like to see this placeholder in the godot editor. Can that be done in a clean way with translations active?

Reply:
It's either there and will be extracted in the translation files, or not there. That's why I removed it since ingame, it is computed anyway.

I removed those to try to reduce as much as possible the number of unnecessary strings in the translation system.

Reply:
It makes editing the GUI easier when there are placeholders that you can see while editing.
That's why in another comment I suggested adding some custom filtering on top to allow these placeholders to be in the scene files. If it's needed anyway for purely numeric stuff, then the filtering is needed anyway and could be used for this as well.

Reply:
We should then decide on a way to mark those string as a placeholder, so that we can filter them when extracting them.

Reply:
Some filter like "Only numbers and math operators" is good for filtering out all the things that shouldn't be translated.
For placeholders, I think we'll just need a big list for them.
One alternative I could think of is to have string IDs starting with PLACEHOLDER_, to tell translators that they don't need to translate them. However, this approach only works if the Godot editor shows the widget text in translated form.
Otherwise the PLACEHOLDER_VERSION is very different length from the actual text used, and doesn't really work well for seeing in the GUI editor what it's going to look like.

Reply:
I updated my fork of babel Godot to ignore as many numbers as possible by default.
And to ignore manually, you simply need to add PLACEHOLDER in the editor description field.

![image](https://user-images.githubusercontent.com/5007519/93023507-c63b8600-f5ef-11ea-8b93-42c34ed74b17.png)

Reply:
That's pretty good. I assume then that everyone will need to install your fork of pybabel for it to work?

Reply:
Yes, but I uploaded it to PyPI so they'll just need to install pip (That they would need to install Babel anyway) and run :
pip install Babel Babel-Thrive

Reply:
Interesting. I suppose you can just add packages like that to PyPI.

Reply:
Yes, you just need to create an account and then you can add packages. The only limitation is to have a unique project name I guess.